code/modules/mob/living/simple_animal/bot/bot.dm

```diff
@@ -462,14 +465,53 @@
 	if(user.a_intent == INTENT_HARM)
 		return ..()
 	if(locked)
-		to_chat(user, "<span class='warning'>The maintenance panel is locked.</span>")
+		to_chat(user, "<span class='warning'>The interface is locked.</span>")
 		return TRUE // Must be true or we attempt to stab the bot
 
 	open = !open
 	I.play_tool_sound(src)
 	to_chat(user, "<span class='notice'>The maintenance panel is now [open ? "opened" : "closed"].</span>")
 	return TRUE
 
+/mob/living/simple_animal/bot/wrench_act(mob/living/user, obj/item/I)
+	if(user.a_intent != INTENT_HELP)
+		return..()
+	if(locked) //ID-Lock
+		to_chat(user, "<span class='warning'>The interface is locked.</span>")
+		return TRUE // Must be true or we attempt to bash the bot
+	if(!open) //Screwdriver panel
+		to_chat(user, "<span class='warning'>The maintenance panel is screwed in place.</span>")
+		return TRUE // Must be true or we attempt to bash the bot
+	if((bot_type != SEC_BOT) && (bot_type != GRIEF_BOT)) //Should only affect security bots and griefsky
+		to_chat(user, "<span class='warning'>The internal components cannot be wrenched more securely.</span>")
+		return TRUE // Must be true or we attempt to bash the bot
+
+	I.play_tool_sound(src)
+	wrenched = !wrenched
+	to_chat(user, "<span class='notice'>The internal components are now [wrenched ? "wrenched" : "unwrenched"].</span>")
+	return TRUE
+
+// Override me please!
+/mob/living/simple_animal/bot/proc/disassemble()
+	qdel(src)
```

Comment:
```suggestion
/mob/living/simple_animal/bot/proc/disassemble()
	CRASH("disassemble() not overridden on [type]")
```

Reply:
Out of curiosity, who has access to the file were the debugging output is logged ? Maintainers and runtime access ?

Reply:
yep, I'm guessing if this pops up in the in-game runtime viewer then someone is going to report it just out of annoyance.

Also this should pop up in local testing if people add a new type and spawn it. By doing this we ensure that future coders will properly override this proc.

Reply:
Okay, thank you for the explanation